File: cicd/gatling/src/test/scala/io/perfana/afterburner/gatling/feeders/CSVFeeder.scala
package io.perfana.afterburner.gatling.feeders


import scala.concurrent.duration._

import io.gatling.core.Predef._
import io.gatling.http.Predef._
import io.gatling.jdbc.Predef._

object CSVFeeder {

  val firstName = csv("data/firstNames.csv").random
  val lastName = csv("data/lastNames.csv").random

}

File: cicd/gatling-kotlin/src/test/kotlin/io/perfana/afterburner/gatling/feeders/CSVFeeder.kt
package io.perfana.afterburner.gatling.feeders

import scala.concurrent.duration.*

import io.gatling.core.Predef.*
import io.gatling.http.Predef.*
import io.gatling.jdbc.Predef.*

/**
 * Created by x077411 on 12/12/2014.
 */
object CSVFeeder {

    val firstName = csv("data/firstNames.csv").random()
}
